test/integration/kubelet/watch_manager_test.go

```diff
@@ -65,7 +65,7 @@ func TestWatchBasedManager(t *testing.T) {
 	// create 1000 secrets in parallel
 	t.Log(time.Now(), "creating 1000 secrets")
 	wg := sync.WaitGroup{}
-	errCh := make(chan error, 1)
+	errCh := make(chan error, 10)
```

Comment:
nit: worth making it a constant as it is used in 4 places that needs to be coordinated.

Reply:
Thanks. Added.